docs/events/add-manage-notes.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[add-manage-notes]]
+= Notes
+
+Incorporate notes into your investigative workflows to coordinate responses, conduct threat hunting, and share investigative findings. You can attach notes to alerts, events, and Timeline and manage them from the **Notes** page. 
+
+NOTE: Configure the `securitySolution:maxUnassociatedNotes` <<max-notes-alerts-events,advanced setting>> to specify the maximum number of notes that you can attach to alerts and events. 
+
+[discrete]
+[[add-notes-documents]]
+== Add notes to alerts and events
+
+Open the alert or event details flyout to access the **Notes** tab, where you can add notes to alerts and events. To quickly open the tab, use the **Add note** action (image:images/create-note-icon.png[Add note action,15,15]) in the Alerts or Events table. Then, enter a note into the text box, and click **Add note** to create it.
+
+[role="screenshot"]
+image::images/create-new-note.png[Creating a new note]
+
+In the alert details flyout, the new note displays on the **Notes** tab. The alert's summary also updates and shows how many notes are attached to the alert. In the event details flyout, the new note displays on the **Notes** tab only.
+
+[role="screenshot"]
+image::images/new-note-added-flyout.png[New note added to an alert]
+
+[discrete]
+[[find-documents-with-notes]]
+=== Find alerts and events with notes
+
+To find alerts and events with notes, use the <<manage-notes,**Notes** page>>. Alternatively, go to the Alerts or Events tables, and look for alerts and events with a notification dot over the **Add note** action (image:images/create-note-icon.png[Add note action,15,15]). Click the action to open the **Notes** tab, which displays all notes that are attached to the alert or event.
+
+[role="screenshot"]
+image::images/notes-notification.png[Notes notification dot on Alerts page]
+
+[discrete]
+[[add-notes-timelines]]
+== Add notes to Timelines
+
+IMPORTANT: You can only add notes to saved Timelines.  
+
+There are two ways to add notes to a saved Timeline: 
+
+* Open the Timeline, go to the **Notes** tab, and create a new note.
+* Open the details flyout for alerts and events that you're investigating in Timeline, and create a new note. Be aware that notes are automatically attached to the Timeline unless you deselect the **Attach to current Timeline** option.
+
+[discrete]
+[[find-timelines-with-notes]]
+=== Find Timelines with notes
```

Comment:
same here, I feel like this section is redundant and a tiny bit confusing, especially second half that basically says: go to the Timelines screen, open each timeline to see if they have notes... This is not a good UX so I don't think we should encourage users to do that.
I guess one day we should update the Timelines table to show which ones have notes and which ones don't... That would be much better!

Reply:
Yep, I can remove this. Directing users to the Notes page to find Timelines with notes is the happier path and better UX.

Reply:
Removed in [b438eae](https://github.com/elastic/security-docs/pull/6006/commits/b438eaef9668518300148f12d0c09d0d1b353c10).